test/e2e/conftest.py

```diff
@@ -40,6 +47,14 @@ def fixture_cleanup_database_collections():
     database.settings.delete_many({})
     # Remove extra usage statuses but leave predefined ones
     database.usage_statuses.delete_many({"_id": {"$nin": PREDEFINED_USAGE_STATUS_IDS}})
+    # Ensure predefined usage statuses have not been accidentally removed and insert them if they have
+    for usage_status in [
+        USAGE_STATUS_OUT_DATA_NEW,
+        USAGE_STATUS_OUT_DATA_IN_USE,
+        USAGE_STATUS_OUT_DATA_USED,
+        USAGE_STATUS_OUT_DATA_SCRAPPED,
+    ]:
```

Comment:
```suggestion
    for usage_status in PREDEFINED_USAGE_STATUS_IDS:
```
Maybe to match the above?

Reply:
But `PREDEFINED_USAGE_STATUS_IDS` is a list of IDs and this the usage statuses to insert them if the IDs do not exist.

Reply:
Yeah it would, its only the fact it doesnt attempt to insert it unless it doesnt exist, it is still however an update replacing the contents if it does. In my first part yeah I should have renamed the variable to `PREDEFINED_USAGE_STATUSES` which just contains the same things (just thought it might be nice to keep the definition of which ones were predefined in the mock data file).

Yeah we could just empty them at that point, then again we would still save repeatedly clearing and inserting all of them which is two separate operations. In this case you could even use upsert with update_many to do all at once.

Reply:
Looks like the only reason it modifies them is because the `created_time` and `modified_time` values differ each time these constants are referenced. At the moment we have 6 different operations - one for `delete_many` and `update_one` for each usage status. This will reduce down to 2 operations - one for `delete_many` and another for `insert_many`